opus/application/static_media/js/opus.js

```diff
@@ -127,30 +128,21 @@ var opus = {
             $(".op-reset-button button").prop("disabled", true);
         }
 
-        // compare selections and last selections
-        if (o_utils.areObjectsEqual(selections, opus.last_selections)) {
-            // selections have not changed from opus.last_selections
+        // compare selections and last selections, extras and last extras
+        if (o_utils.areObjectsEqual(selections, opus.lastSelections) && o_utils.areObjectsEqual(extras, opus.lastExtras)) {
+            // selections have not changed from opus.lastSelections
```

Comment:
This comment should be updated to include extras

Reply:
Comments are updated to include extras.